config/bf.py

```diff
@@ -0,0 +1,8 @@
+config.isr.doFlat = False
+config.isr.doFringe = False
+config.isr.doLinearize = False
+config.isr.doAddDistortionModel = False
+config.isr.doUseOpticsTransmission = False
+config.isr.doUseFilterTransmission = False
+config.isr.doUseSensorTransmission = False
+config.isr.doUseAtmosphereTransmission = False
```

Comment:
I think the task `DefaultName` and thus this file name should be "measureBrighterFatter" or "measureBrighterFatterKernel".

Also, this is a lot of overrides...can these config parameters be set as defaults by the "measure brighter fatter" task itself?

Reply:
Good call on the config filename, thanks. I don't think the defaults can necessarily be set in the task itself, that seems like dangerous territory, but they are checked to be correct if the keys are present in the config dict; I hope this will suffice (and this is the same for all packages).

Reply:
Having updated the filename I'll assume I'm OK to merge unless you shout.